sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -233,7 +233,7 @@ namespace Azure { namespace Security { namespace Attestation {
     /**
      * @brief Sends TPM-based attestation data to the service.
      * The TPM attestation protocol is defined
-     * [here](https://docs.microsoft.com/azure/attestation/virtualization-based-security-protocol')
+     * [here](https://learn.microsoft.com/azure/attestation/virtualization-based-security-protocol')
```

Comment:
nit: remove the back-tick at the end (it doesn't have a matching begin anyway)? I know it was there before, but looks like a typo and if someone copy-pastes the whole link (including the back-ttick), it will be 404

```suggestion
     * [here](https://learn.microsoft.com/azure/attestation/virtualization-based-security-protocol)
```

Reply:
applied suggestion